conda/base/context.py

```diff
@@ -1396,6 +1423,28 @@ def description_map(self):
                 into the first writable directory.
                 """
             ),
+            prerelease_behavior=dals(
+                """
+                Specifies how to handle prerelease versions (e.g. 1.2.3.dev3) when installing
+                or updating. The options are 'allow' (the default), 'limit', and 'exclude'.
+
+                When 'allow' is selected, prerelease versions are treated the same as any
+                other version.
+
+                When 'limit' is selected, prerelease packages will only be used if already
+                present or is the only way to satisfy a dependency.
```

Comment:
I might have missed this, but where are we checking whether a given package is already using a pre-release version and thus pre-release updates should be allowed? This sounds like the trickiest mode to implement, so we need tests covering these cases specifically.